apps/dh/api-dh/source/DataHub.WebApi/Clients/EDI/B2CWebApi/V1/swagger.json

```diff
@@ -0,0 +1,103 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "B2C web api for EDI",
+    "version": "v1"
+  },
+  "paths": {
+    "/RequestAggregatedMeasureData": {
+      "post": {
+        "tags": [
+          "RequestAggregatedMeasureData"
+        ],
+        "operationId": "RequestAggregatedMeasureData",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/RequestAggregatedMeasureDataMarketRequest"
+              }
+            },
+            "text/json": {
+              "schema": {
+                "$ref": "#/components/schemas/RequestAggregatedMeasureDataMarketRequest"
+              }
+            },
+            "application/*+json": {
+              "schema": {
+                "$ref": "#/components/schemas/RequestAggregatedMeasureDataMarketRequest"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "MeteringPointType": {
+        "type": "integer",
+        "format": "int32",
+        "enum": [
+          0,
+          1,
+          2,
+          3,
+          4
+        ]
+      },
+      "RequestAggregatedMeasureDataMarketRequest": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "businessReason": {
+            "type": "string",
+            "nullable": true
+          },
+          "meteringPointType": {
+            "$ref": "#/components/schemas/MeteringPointType"
+          },
+          "startDate": {
+            "type": "string",
+            "nullable": true
```

Comment:
According to the c# class, neither this and businessReason at line 58 is nullable?